README.md

```diff
@@ -60,3 +60,31 @@ Press CTRL+C to quit
  * Debugger PIN: 113-666-066
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing the App
+### Dependencies
+
+Install pytest by running:
+
+```bash
+$ poetry add pytest
+```
```

Comment:
I know why people always want to document this, but worth noting that since this adds the library to your `pyproject.toml` and `poetry.lock` files, future developers can ignore this and they'll just pick it up as part of `poetry install`

Reply:
Haha good point! 😁